drafts/2019-11-06-1900011.adoc

```diff
@@ -157,6 +160,95 @@ MetadataBuilder mdb = Metadata.builder();
+** Depending on Jaeger's sampling settings `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`, Jaeger may not report every spans created by the applications.
+
+[#LAC]
+== Liberty Annotation Caching
+
+During application startup, application classes are scanned for class signature information and for annotations. Typically, all application classes are scanned, which is time consuming.
+
+For early access, Annotation caching is enabled when the product edition is `EARLY_ACCESS`.
+
+For other product editions, annotation caching is disabled, but may be forced on by setting java custom property "anno.beta" to "true".
+
+When the product edition is `EARLY_ACCESS`, the caching function may be forced off by setting java custom property `anno.beta` to `false`.
+
+The new scan engine may be enabled with cache function turned off by setting java custom property `anno.cache.disabled` to `true`.
+
+The new scan engine may be prevented from writing new data by setting java custom property `anno.cache.readOnly` to `true`.  Already generated cache data will still be read.  This may be necessary in
+certain deployment pipelines.
+
+When enabled, application class scan steps automatically write (cache) scan data, and automatically read cachedscan data.  Cache data is automatically regenerated when changes are detected in application classes.  No
+additional settings are necessary to use the new function.  (Pipeline developers, however, see "Important Note", below.)
+
+Cache data is visible as a new, nested directory of server workarea named `anno`.  (Cache data uses a binary
+format and is not human readable.)  Cache data is relocatable: Movement of a server image to a new location
+does not require that the application class scan cache data be regenerated.
+
+The cache of application class data is cleared when performing a clean server start (starting the server with
+the "--clean" option).  In normal operations, the clearing of cache data is not necessary, since the cache 
+automatically detects class changes and regenerates invalid cache data.  However, the cache function has no
+way to tell when cache data is obsolete.  If the same server is used with many different applications, a clean
+start should be performed occasionally to remove stale cache data.
+
+Important Note: In a pipeline deployment environment, for caching to be effective, the server image must be
+"warmed" before packaging the deployment image, and application class cache data must be included in the
+deployment image.  "Warming" the server means performing a single start of the server.  This will cause
+cache data to be generated and available when packaging the server image.  A failure to generate the cache
+data or to include the cache data in the deployment image prevents the server deployment from obtaining
+the full value of the new caching function.
+
+This chart shows changes to server startup times comparing server startups which do not use the new caching function
```

Comment:
Find the chart image and include it straight after this paragraph. Change the wording to "The following chart..."